LCM/gen/dsccore/MSFT_DSCMetaConfiguration.mof

```diff
@@ -11,8 +11,8 @@ class MSFT_DSCMetaConfiguration
    boolean RebootNodeIfNeeded;
 
    [Description ("The configuration apply mode for the server."),
-   ValueMap {"ApplyOnly","ApplyAndMonitor","ApplyAndAutoCorrect"},
-   Values {"ApplyOnly","ApplyAndMonitor","ApplyAndAutoCorrect"}]
+   ValueMap {"ApplyOnly","ApplyAndMonitor","ApplyAndAutoCorrect", "MonitorOnly"},
+   Values {"ApplyOnly","ApplyAndMonitor","ApplyAndAutoCorrect", "MonitorOnly"}]
```

Comment:
>ct", "MonitorOn [](start = 59, length = 15)

do we need change in schema.c as well?

Reply:
i think we don't. 

---
In reply to: [138499994](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/311#discussion_r138499994) [](ancestors = 138499994)